67_RayQueryGeometry/common.hlsl

```diff
@@ -0,0 +1,25 @@
+#ifndef RQG_COMMON_HLSL
+#define RQG_COMMON_HLSL
+
+NBL_CONSTEXPR uint32_t WorkgroupSize = 16;
+
+#ifdef __HLSL_VERSION
+struct VertexData
+{
+    float3 position;
+    float4 color;
+    float2 uv;
+    float3 normal;
+};
```

Comment:
with modern rendering you REALLY don't want to use interleaved vertex formats

Reply:
hmmm I see that Geometry Creation spits out interleaved, in that case don't bother de-interleaving